docs/joss_paper/paper.md

```diff
@@ -82,35 +82,35 @@ The loss functions described in the previous sections are often combined with a
 
 ## The software
-`DeepReg` is unit tested, and includes both technical documentation and various tutorials to introduce users to various use cases using open-sourced datasets. The continuous integration platform TravisCI (https://travis-ci.org/) is used together with a cutomised contributor guideline to facilitate contributions.
+`DeepReg` is unit tested, and includes both technical documentation and various tutorials to introduce users to various use cases, using openly-accessible datasets. The continuous integration platform TravisCI (https://travis-ci.org/) is used together with a cutomised contributor guideline to facilitate community contributions.
 
 
-# Example applications
+# DeepReg Demos
 `DeepReg` provides a collection of demonstrations, _DeepReg Demos_, using open-accesible data with real clinical applications.
 
 ## Paired images
-If images are available in pairs, two dataset loaders are currently available for paired images with- and without labels. Many clinical application for tracking organ motion and other temporal changes between two time points require _intra-subject_ _single-modality_ image registration. Registering lung CT images for the same patient, acquired at expritory and inspritory, is such an example of both unsupervised (without labels) and combined supervision (with segmentation of anatomical structures). Furthermore, registering prostate MR, aquired before surgery, and the intra-operative trasnrectal ultrasound images is an example of weakly-supervised training for multimodal image registration.
+If images are available in pairs, two dataset loaders are currently available for paired images with- and without labels. Many clinical application for tracking organ motion and other temporal changes between two time points require _intra-subject_ _single-modality_ image registration. Registering lung CT images for the same patient, acquired at expiratory and inspiratory, is such an example of both unsupervised (without labels) and combined supervision (trained wwith label dissimilarity based on segmentation of anatomical structures). Furthermore, registering prostate MR, aquired before surgery, and the intra-operative trasnrectal ultrasound images is an example of weakly-supervised training for multimodal image registration. Another DeepReg Demo of MR-to-ultrasound image registration is to track brain tumour locations during neurosurgery, in which combining unsupervised and weakly-supervised losses may be useful.
```

Comment:
trasnrectal -> transrectal
with- and without -> why the hyphen?
Another DeepReg Demo of MR-to-ultrasound image registration is to track brain tumour locations during neurosurgery, -> Another DeepReg Demo illustrates MR-to-ultrasound image registration for localization of brain tumours in neurosurgery,